bool detection
string name
float32 dtheta_x
float32 dtheta_y
float32 dtheta_z